src/bridge/cockpitpipechannel.c

```diff
@@ -401,7 +461,7 @@ cockpit_pipe_channel_prepare (CockpitChannel *channel)
                                 "invalid \"directory\" option for stream channel");
           goto out;
         }
-      if (!cockpit_json_get_bool (options, "pty", FALSE, &pty))
+      if (!cockpit_json_get_bool (options, "pty", FALSE, &self->pty))
```

Comment:
I actually did drop it, but in a later commit. Moved it into the one that it belongs to. Thanks!